programs/mpl-core/src/instruction.rs

```diff
@@ -127,15 +127,15 @@ pub(crate) enum MplAssetInstruction {
     #[account(0, writable, name="asset", desc = "The address of the asset")]
     #[account(1, optional, writable, name="collection", desc = "The collection to which the asset belongs")]
     #[account(2, writable, signer, name="payer", desc = "The account paying for the storage fees")]
-    #[account(3, optional, signer, name="authority", desc = "The owner or delegate of the asset")]
+    #[account(3, optional, writable, signer, name="authority", desc = "The owner or delegate of the asset")]
     #[account(4, optional, name="system_program", desc = "The system program")]
     #[account(5, optional, name="log_wrapper", desc = "The SPL Noop Program")]
     BurnV1(BurnV1Args),
 
     /// Burn an mpl-core.
     #[account(0, writable, name="collection", desc = "The address of the asset")]
     #[account(1, writable, signer, name="payer", desc = "The account paying for the storage fees")]
-    #[account(2, optional, signer, name="authority", desc = "The owner or delegate of the asset")]
+    #[account(2, optional, writable, signer, name="authority", desc = "The owner or delegate of the asset")]
     #[account(3, optional, name="log_wrapper", desc = "The SPL Noop Program")]
```

Comment:
I think the payer is the default receiver of the funds and if authority is specified it is the receiver.  So I think payer must be set to writable here.

Reply:
so according to the code, it's the authority that receives the rent, which may or may not be the payer. Did we need to change the logic to always be the payer (I don't think so?)

Also the payer is already marked as writable if that is what your comment is referring to

Reply:
No it's fine. Somehow I looked at it and did not see payer was already writable and thought it wasn't